Environment/NoiseLevelObserved/doc/spec.md

```diff
@@ -40,15 +40,15 @@ This entity is primarily associated with the Smart City and environment vertical
 It can be represented by an specific time instant or by an ISO8601 interval. As a workaround for
 the lack of support of Orion Context Broker for datetime intervals, it can be used two separate attributes: `dateObservedFrom`, `dateObservedTo`. 
     + Attribute type: [DateTime](https://schema.org/DateTime) or an ISO8601 interval represented as [Text](https://schema.org/Text). 
-    + Mandatory
+    + Optional
         
 + `dateObservedFrom` : Observation period start date and time. See `dateObserved`. 
```

Comment:
Regarding to dateObservedFrom and dateObservedTo, I would say that they can be optional but not mandatory. Observed values may have a timestamp metadata stating the date the observation was adquired. It could involve that dateObservedFrom and dateObservedTo attributes to be not so relevant. My proposal is to state them as Optional.

Reply:
Be careful. Noise level measurement always is performed during an integration period, so it makes sense to make these parameters mandatory.